build/target-repository/composer.json

```diff
@@ -2,6 +2,7 @@
     "name": "rector/rector",
     "description": "Instant Upgrade and Automated Refactoring of any PHP code",
     "license": "MIT",
+    "keywords": ["dev", "refactoring", "automation", "migration"],
```

Comment:
only in interactive sessions, e.g. in CI you should run non-interactively already

Reply:
e.g. phpstan is using it for a few months already

Reply:
As @samsonasik write, Rector can be both dev tool and core package for any extension, so using `dev` is not true here.

Reply:
rector works pretty similar as phpstan in this regard, so it should not be a problem.

we only need to make sure we add the keyword in the correct composer.json.

I am not 100% sure, which are the right ones in rector repos though, because there are a few of them.
maybe we just need to add it directly in `rector/rector`, nowhere else?

Reply:
Looking at the composer PR implementation, that seems only on `RequireCommand`, if the package already in "require" in composer.json, running composer install/update still show the prompt?

Reply:
you can try any combination with `phpstan/phpstan`, as it has this keyword already defined.